core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7475,6 +7478,15 @@ components:
                 because the blueprint code relies on the total supply of the pool units to calculate their
                 redemption worth, and you've just inflated the total supply through the mint operation.
               type: boolean
+    TransactionPreviewRequestOptIns:
+      type: object
+      properties:
+        request_radix_engine_toolkit_receipt:
```

Comment:
For consistency, we probably just want `radix_engine_toolkit_receipt`

Reply:
Sounds good! Making this change